src/DurableTask.Core/FailureDetails.cs

```diff
@@ -51,6 +51,17 @@ public FailureDetails(Exception e)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FailureDetails"/> class from an exception object and
+        /// an explicitly specified inner exception.
+        /// </summary>
+        /// <param name="e">The exception used to generate the failure details.</param>
+        /// <param name="InnerException">The inner exception to use.</param>
+        public FailureDetails(Exception e, Exception InnerException)
```

Comment:
nit: constructor parameters should be lowercase.

```suggestion
        public FailureDetails(Exception e, Exception innerException)
```

Reply:
yes.